src/components/Global/QRScanner/index.tsx

```
@@ -65,77 +54,222 @@ function PaymentMethodBadge({ src, alt, name }: { src: string; alt: string; name
+    const videoRef = useRef<HTMLVideoElement>(null)
+    const canvasRef = useRef<HTMLCanvasElement>(null)
+    const scanIntervalRef = useRef<number | null>(null)
+    const streamRef = useRef<MediaStream | null>(null)
+    const retryCountRef = useRef<number>(0)
+    const stopCamera = useCallback(() => {
+        if (scanIntervalRef.current) {
+            clearInterval(scanIntervalRef.current)
+            scanIntervalRef.current = null
+        }
+        if (streamRef.current) {
+            streamRef.current.getTracks().forEach((track) => track.stop())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)